src/main/fc/rc.c

```diff
@@ -231,6 +233,7 @@ float getRcCurveSlope(int axis, float deflection)
 static void calculateSetpointRate(int axis)
 {
     float angleRate;
+    float rcCommandf;
```

Comment:
Declarations at the beginning of a function are discouraged in Betaflight, as they make code harder to read and maintain. Please move this back.

Reply:
ok